lib/bundler/ruby_version.rb

```diff
@@ -114,4 +94,32 @@ def patchlevel
       RUBY_PATCHLEVEL.to_s
     end
   end
+
+  class RubyVersionRequirement < RubyVersion
```

Comment:
I'm starting to feel that this file should be broken down into several. It's already defining 3 classes :disappointed: